### PR TITLE
README.md: Add Windows support and mention MetaMask issue

### PR DESCRIPTION
**Pull request by caovanthanh203**
Add alternative command to export env variable for Windows

### CODE REVIEW

**Comment by matevz** [comment]
Can you check, if other commands (pnpm, npx, hardhat) for deployment and setMessage work as expected?

**Comment by bc67da8d** [comment]
> Can you check, if other commands (pnpm, npx, hardhat) for deployment and setMessage work as expected?

Yeah, other commands works, I am using Windows so I can confirm they are good with CMD

File: frontend/.env.development [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# VITE_NETWORK=1337 #localhost
```
Comment (matevz) [inline frontend/.env.development:1]: Please remove lines of code that are not relevant to this PR.

```suggestion
```

File: README.md [hunk 2]
```diff
@@ -75,6 +81,13 @@ Hardhat Node, Foundry Anvil or sapphire-dev docker image, don't forget to
 *clear your account's activity* each time or manually specify the correct
 account nonce.
 
+When click button Connect Wallet, in some case you will constantly get the error: 
+**Uncaught (in promise) Error: [useEthereumStore] Request account failed!**, 
+
+To resolve it, try to open MetaMask and manually connect site once time
+
+![alt text](manually-connect-site-1.png)  ![alt text](manually-connect-site-2.png)
+
```
Comment (matevz) [inline README.md:90]: I cannot reproduce this one and it seems as a MetaMask-specific error. I would either not put this into README.md at all or add a new `## Troubleshooting` at the end of the README.md with the description above.

Please place the two screenshots into the new docs/ folder.

**Review by matevz — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (caovanthanh203) [reply README.md]: I added a new ## Troubleshooting at the end of the README.md with the description above & moved two screenshots into the new docs/ folder

File: README.md [hunk 3]
```diff
@@ -100,3 +106,12 @@ pnpm build-only --base=/my/public/path/
...
 Then copy the `dist` folder to a place of your `/my/public/path` location.
+
+## Troubleshooting
+
+When click button Connect Wallet, in some case you will constantly get the error: 
+**Uncaught (in promise) Error: [useEthereumStore] Request account failed!**, 
+
+To resolve it, try to open MetaMask and manually connect site once time
+
+![alt text](docs/manually-connect-site-1.png)  ![alt text](docs/manually-connect-site-2.png)
```
Comment (matevz) [inline README.md:117]:
```suggestion
![MetaMask: Connected sites](docs/manually-connect-site-1.png)
![MetaMask: Manually connect to the current site](docs/manually-connect-site-2.png)
```

File: README.md [hunk 4]
```diff
@@ -100,3 +106,12 @@ pnpm build-only --base=/my/public/path/
 ```
 
 Then copy the `dist` folder to a place of your `/my/public/path` location.
+
+## Troubleshooting
+
+When click button Connect Wallet, in some case you will constantly get the error: 
+**Uncaught (in promise) Error: [useEthereumStore] Request account failed!**, 
+
+To resolve it, try to open MetaMask and manually connect site once time
```
Comment (matevz) [inline README.md:115]:
```suggestion
To resolve it, try to open MetaMask and manually connect site:
```

File: README.md [hunk 5]
```diff
@@ -100,3 +106,12 @@ pnpm build-only --base=/my/public/path/
 ```
 
 Then copy the `dist` folder to a place of your `/my/public/path` location.
+
+## Troubleshooting
+
+When click button Connect Wallet, in some case you will constantly get the error: 
+**Uncaught (in promise) Error: [useEthereumStore] Request account failed!**,
```
Comment (matevz) [inline README.md:113]:
````suggestion
`Uncaught (in promise) Error: [useEthereumStore] Request account failed!`
````

**Review by matevz — CHANGES_REQUESTED** [review]
Some nits.

**Review by matevz — APPROVED** [review]
Looks good, thanks!

**Pull request merged**